src/ghvmctl

```diff
@@ -10,6 +10,13 @@ VM_CPUS="${VM_CPUS:=1}"
 VM_MEM_GIB="${VM_MEM_GIB:=6}"
 VM_DISK_GIB="${VM_DISK_GIB:=12}"
 
+REAL_HOME=${SNAP_REAL_HOME:-$PWD}
+GHVMCTL_RUNNER_PATH=$(dirname "$0")
+
+if [ -v SNAP ]; then
+	GHVMCTL_RUNNER_PATH=${SNAP}/opt/ghvmctl
+fi
```

Comment:
```suggestion
```

If you accept the suggestion above, this can be removed I think.

Reply:
Fair, I'd still need the check for `REAL_HOME`?

I'm fine using another place, even pure `$HOME` there though, as you prefer.

Reply:
Sounds good to me :)